packages/test/test-end-to-end-tests/.eslintrc.js

```diff
@@ -4,14 +4,28 @@
  */
 
 module.exports = {
-    "extends": [
-        require.resolve("@fluidframework/eslint-config-fluid")
-    ],
-    "rules": {
+    extends: [require.resolve("@fluidframework/eslint-config-fluid")],
+    rules: {
         "prefer-arrow-callback": "off",
         "@typescript-eslint/strict-boolean-expressions": "off", // requires strictNullChecks=true in tsconfig
+
+        // This library is used in the browser, so we don't want dependencies on most node libraries.
+        "import/no-nodejs-modules": ["error", { allow: ["url"] }],
     },
-    "parserOptions": {
-        "project": [ "./src/test/tsconfig.json" ]
-    }
-}
+    parserOptions: {
+        project: ["./src/test/tsconfig.json"],
+    },
+    overrides: [
+        {
+            // Rules only for test files
+            files: ["*.spec.ts", "src/test/**"],
+            rules: {
+                // This library is used in the browser, so we don't want dependencies on most node libraries.
```

Comment:
Nit: maybe "Test files are run in node only so additional node libraries can be used.